api_app/analyzers_manager/file_analyzers/mobsf_service.py

```diff
@@ -0,0 +1,128 @@
+import logging
+
+import requests
+
+from api_app.analyzers_manager.classes import FileAnalyzer
+from tests.mock_utils import MockUpResponse, if_mock_connections, patch
+
+logger = logging.getLogger(__name__)
+
+
+class MobSF_Service(FileAnalyzer):
+    mobsf_host: str
+    _mobsf_api_key: str
+
+    def update(self) -> bool:
+        pass
+
+    def static_analysis(self, scan_hash, headers):
+
+        scan_url = self.mobsf_host + "/api/v1/scan"
+        data = {"hash": scan_hash}
+        scan_response = requests.post(url=scan_url, data=data, headers=headers)
+        scan_response.raise_for_status()
+
+        report_url = self.mobsf_host + "/api/v1/report_json"
+        report_response = requests.post(url=report_url, data=data, headers=headers)
+        report_response.raise_for_status()
+        return report_response.json()
+
+    def dynamic_analysis(self, scan_hash, headers):
+
+        mobsfy_runtime_response = requests.post(
+            url=self.mobsf_host + "/api/v1/android/mobsfy",
+            headers=headers,
+            data={"identifier": "127.0.0.1:6555"},
+        )
+        mobsfy_runtime_response.raise_for_status()
+
+        start_dynamic_analysis_response = requests.post(
+            url=self.mobsf_host + "/api/v1/dynamic/start_analysis",
+            headers=headers,
+            data={"hash": scan_hash},
+        )
+        start_dynamic_analysis_response.raise_for_status()
+
+        tls_tests_response = requests.post(
+            url=self.mobsf_host + "/api/v1/android/tls_tests",
+            headers=headers,
+            data={"hash": scan_hash},
+        )
+        tls_tests_response.raise_for_status()
+
+        start_frida_instrumentation_response = requests.post(
+            url=self.mobsf_host + "/api/v1/frida/instrument",
+            headers=headers,
+            data={
+                "hash": scan_hash,
+                "default_hooks": "api_monitor,ssl_pinning_bypass,root_bypass,debugger_check_bypass",
+                "auxiliary_hooks": "",
+                "frida_code": "",
```

Comment:
these 3 values should be parameters of the analyzer. In that way the users can choose how to leverage dynamic analysis as they wish

Reply:
Sure I'll add them as parameters for users to pass a desired value.